src/card.css

```
@@ -0,0 +1,111 @@
+.card {
+  background: var(--clear_white-color);
+  border: 1px solid #f3f3f3;
```

create vars for all colors
```suggestion
  border: 1px solid var(--some-color);
```

---

fixed